### PR TITLE
Sommelier-18

### PR DESCRIPTION
**Pull request by constantkek**

### CODE REVIEW

**Comment by constantkek** [comment]
checked: image
unchecked: image
knobs: 
image

**Comment by constantkek** [comment]
checked:
image
unchecked:
image
knobs:
image

File: App/components/Checkbox/checkbox.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React, { ReactNode } from 'react'
+import { StyleSheet } from 'react-native'
+import NativeCheckbox from '@react-native-community/checkbox'
+import { CheckboxContainer, CheckboxText } from './checkbox.styled'
+
+type Props = {
+  checked: boolean
```
Comment (Mixxx-a) [inline App/components/Checkbox/checkbox.tsx:7]: Shouldn't this prop be optional?

File: App/components/Checkbox/checkbox.styled.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components/native'
+
+export const CheckboxContainer = styled.View`
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+`
+
+export const CheckboxText = styled.Text`
+  color: ${({ theme }) => theme.colors.black};
```
Comment (Mixxx-a) [inline App/components/Checkbox/checkbox.styled.ts:10]: Black is the only color, or should color be a one from the theme?

File: App/components/Checkbox/checkbox.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  children: ReactNode
+}
+
+const Checkbox = ({ checked, children }: Props) => (
+  <CheckboxContainer>
+    <NativeCheckbox
+      value={checked}
+      tintColors={{ true: '#A22841', false: '#757575' }}
+    />
+    <CheckboxText>{children}</CheckboxText>
```
Comment (Mixxx-a) [inline App/components/Checkbox/checkbox.tsx:17]: In children prop we can pass any component(s) that we want?

**Review by Mixxx-a — COMMENTED** [review]
Will create tests for that component

**Review by Mixxx-a — APPROVED** [review]

**Review by StanislavMyakishev — APPROVED** [review]

**Pull request merged**